recipes/appimage-updater-bridge/test.py

```diff
@@ -0,0 +1,9 @@
+import sys
+from PySide2 import QtCore
+
+loader = QtCore.QPluginLoader()
+loader.setFileName("libAppImageUpdaterBridge")
```

Comment:
You need to move the plugin to Qt's default plugin directory. Or mention it with absolute path.

Reply:
@looooo you can  pass the absolute path to the plugin as an environmental variable and grab it inside the python script to test..

Reply:
See https://github.com/antony-jr/AppImageUpdaterBridge/blob/f8eb89d409e08ed912e6efdef7cd0dda2f93352d/examples/PyQt5Update/Update.py#L19

Reply:
I had no issues testing this locally... I think we install the plugin to the right directory.

Reply:
You can just use the absolute path, it will be much easier than figuring out the plugin directory used by PySide.

Reply:
actually I wanted to test if the plugin is installed in the right directory. Let's see if absolute-paths work.

Reply:
Seems like a conda specific bug. @looooo Can we place a dummy test python file?

Reply:
Or if you have time then try finding the plugin path for conda. You can use ```QT_DEBUG_PLUGINS``` for more debugging information which can show the plugin directory.

```
 $ QT_DEBUG_PLUGINS=1 python test.py # With relative paths.
```

Reply:
See https://forum.qt.io/topic/93085/qt-python-development-pyside2/9

Reply:
absolute-path is also failing...

Reply:
testing locally (with docker) there is no problem with relative / absolute-paths. I have no idea.

Reply:
There is a syntax error in the test.py, We have to see qt debug messages to find the cause.

Reply:
@looooo The latest debug build shows that it failed to load because it cannot load libGL.so.1, IIRC we need to install ```libgl1-mesa-dev``` package in ubuntu. Any ideas on getting that package? (https://packages.ubuntu.com/xenial/libgl1-mesa-dev)

Reply:
sure, this is done with cdt's. Let's see if this works now.

Reply:
See https://github.com/conda-forge/pygridgen-feedstock/issues/10 ,but no need to hurry. This can wait.

Reply:
@looooo But one way to solve this is to exclude all GUI capabilities of the library. If you can, Can you wait for 2.x?

Reply:
ah sure, you are right, we need additional yum-requirements for qt-dependent recipes. But I will wait for 2.x with this feedstock. We have 1.8 available via the freecad/label/dev channel, so no hurry.

Reply:
Finally the build succeeds.

Reply:
I guess we can merge right now, and add the discussed changes later.